modules/bullet/space_bullet.h

```diff
@@ -36,6 +36,7 @@
 #include "godot_result_callbacks.h"
 #include "rid_bullet.h"
 #include "servers/physics_server_3d.h"
+#include <vector>
```

Comment:
And again.

Reply:
Master branch has LocalVector, which is more similar to std::vector, you could backport to to 3.2 if you want.

Reply:
Removed `std::vector` in favour of the recently added `LocalVector`.